backend/de.metas.externalreference/src/main/java/de/metas/externalreference/ExternalIdentifier.java

```diff
@@ -35,6 +36,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static de.metas.util.Check.isEmpty;
+
 @Value
 public class ExternalIdentifier
```

Comment:
please clarify the relation between this class and IdentifierString. If this class for v2 and identifierstring for v1?
but i found (just) one usage of IdentifierString in `de.metas.rest_api.v2.bpartner.BpartnerRestControllerTest#createOrUpdateBankAccount_create`

Reply:
I missed that, will fix it.